cmd/openfga/main_test.go

```diff
@@ -569,7 +569,130 @@ func connect(t *testing.T, tester OpenFGATester) *grpc.ClientConn {
 }
 
 func GRPCWriteTest(t *testing.T, tester OpenFGATester) {
+	type output struct {
+		errorCode    codes.Code
+		errorMessage string
+	}
+
+	conn := connect(t, tester)
+	defer conn.Close()
+
+	client := openfgav1.NewOpenFGAServiceClient(conn)
+	resp, err := client.CreateStore(context.Background(), &openfgav1.CreateStoreRequest{
+		Name: "openfga-demo",
+	})
+	require.NoError(t, err)
+	storeID := resp.GetId()
+
+	model := parser.MustTransformDSLToProto(`model
+	schema 1.1
+type user
+
+type document
+  relations
+	define viewer: [user]
+`)
+
+	writeModelResp, err := client.WriteAuthorizationModel(context.Background(), &openfgav1.WriteAuthorizationModelRequest{
+		StoreId:         storeID,
+		TypeDefinitions: model.GetTypeDefinitions(),
+		Conditions:      model.GetConditions(),
+		SchemaVersion:   typesystem.SchemaVersion1_1,
+	})
+	require.NoError(t, err)
+	require.NotPanics(t, func() { ulid.MustParse(writeModelResp.GetAuthorizationModelId()) })
+	modelID := writeModelResp.GetAuthorizationModelId()
+
+	tests := []struct {
+		name   string
+		input  *openfgav1.WriteRequest
+		output output
+	}{
+		{
+			name: "happy-path",
```

Comment:
Please use underscores as it is easier to find failed tests. We've defaulted to `_` where we can.